cast/js/src/testFixtures/java/com/ibm/wala/cast/js/test/TestSimpleCallGraphShape.java

```diff
@@ -1071,4 +1071,38 @@ public void testComplexFinally()
 
     CAstCallGraphUtil.dumpCG(B.getCFAContextInterpreter(), B.getPointerAnalysis(), CG);
   }
+
+  private static final Object[][] assertionsES6Args =
```

Comment:
For the test cases where the template substitution has a function call, can we assert that the appropriate call edge(s) are present?